docs/faq.md

```diff
@@ -118,18 +135,59 @@ Yes, this is the plan - to use a HWW like Coldcard/Trezor with Specter DIY, with
+
+# USAGE
 
 ## *How do I run the app?*
 
 After following [these steps](https://github.com/cryptoadvance/specter-desktop#how-to-run) 
 You should be able to view it in a browser at: 127.0.0.1:25441/
 If not, see [Troubleshoot](https://github.com/cryptoadvance/specter-desktop/new/master/docs#troubleshoot)
 
+## *What types of ways can I run specter-desktop?*
+
+There are many ways how to run Specter:
+- Specter on local computer, node on remote
+- Specter on a remote node, web interface in local network or over tor (but hardware wallets need to be connected to the node where Specter is running)
+- Specter on a remote node, another Specter on your computer in "hwibridge" mode that gives access to your hardware wallets from the remote node (configurable whitelist)
+
+It depends on your setup, and can be customized accordingly.
+
+Specter-desktop makes many requests to bitcoin RPC, so it works better from the same machine where Core is running, but remote is also possible. With that being said, by default bitcoin rpc is connecting over http, so everything including your RPC login and password are flying around as plaintext. You can use https and a [self-signed certificate](https://github.com/cryptoadvance/specter-desktop/blob/master/docs/self-signed-certificates.md) to fix that.
+
+If you use hardware wallets and they are usb-conneced to specter-desktop then you should take these precautions, but if they are airgapped (coldcard, specter-diy, cobo) - then you can use remote web interface.
 
 ## *What do I need to do in order to create a multisig wallet?*
 
-First you need to “add devices” that store keys for the wallet. After creating the devices, you have to create the type of wallet you want (2-of-2, 3-of-5, etc.) and select the corresponding devices/keys - you need at least two devices setup in order to create a multisig wallet.
+XPUBs are needed (from HWW's, laptop with Electrum desktop wallet, specter-DIY, etc.) in order to create a multistig setup, but don't worry it's in watch-only mode and it's your own full node! First you need to “add devices” that store keys for the wallet. After creating the devices, you have to create the type of wallet you want (2-of-2, 3-of-5, etc.) and select the corresponding devices/keys - you need at least two devices setup in order to create a multisig wallet.
+
+## *Is my understanding correct that specter-desktop does not hold any keys and you need to create a multisig wallet in order to sign transactions and send funds?*
+
+As of late, you can also use a hot wallet as a signer with specter-desktop, but since it is so new it is not recommended. You can however use devices like Electrum wallet or FullyNoded for example (Electrum or Bitcoin Core can be air-gapped). This [video](https://youtu.be/4YXklLh2srA) is quite useful for using Electrum, and this [guide](https://github.com/Fonta1n3/FullyNoded/blob/master/Docs/Connect-node.md#importing-a-wallet-from-specter) is useful for connecting with FullyNoded.
```

Comment:
The link for the FullyNoded guide is broken.

Reply:
Hmm, looks like the link was changed recently. This one works: https://github.com/Fonta1n3/FullyNoded/blob/master/Docs/Wallets/Specter.md